src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
```

Comment:
You can format the jsx files using Prettier so that it looks better to reviewers